docs/beams/guides/handle-incoming-notifications/web.md

```diff
@@ -58,3 +58,33 @@ PusherPushNotifications.onNotificationReceived = ({
   pushEvent.waitUntil(handleNotification(payload));
 };
 ```
+
+## Accessing Beams state
+
+If you would like to access the Beams state, you can subscribe to the `statePromise` promise in your `onNotificationReceived` handler. For example, you can customize the notification based on the current user's role.
+
+```js
+PusherPushNotifications.onNotificationReceived = async ({
+  pushEvent,
+  payload,
+  handleNotification,
+  statePromise,
+}) => {
+  const { userId } = await
+  pushEvent.waitUntil(
+    // custom notification handling logic
+  );
+};
+```
```

Comment:
Is something missing here? Maybe on line 73.
I would expect this example to include something interacting with the `statePromise`.

Reply:
Oops you're right, it should be `await statePromise`

Reply:
Fixed it, thanks!